### ISSUE TITLE
iOS supported?

### ISSUE THREAD

**Issue by ManigandanRaamanathan** [ShaMan123/react-native-sketch-canvas#12]
@ShaMan123 may I know if iOS is supported? or can you explain the issues existing in iOS compatibility of the current library?

**Comment by ShaMan123** [comment]
#5 
iOS needs work, not yet compatible on `@next` version

**Comment by niyatiArunk** [comment]
is ios will work now ??

**Comment by ShaMan123** [comment]
Hi there,
`react-native-reanimated` is about to release a major version, once they do iOS can be implemented easily

**Comment by bottle-rocket-11** [comment]
Great stuff, I can't wait for this. We use it for a client but still on the original and want to move over to your library but require iOS to work. Will subscribe to this issue. Thank you, your work is appreciated.

**Comment by colelogan** [comment]
@ShaMan123 have you figured ios out yet?

**Comment by ShaMan123** [comment]
Unfortunately not. It isn't too much work but I don't have a mac....
If you'd PR I'll assist where possible

**Comment by colelogan** [comment]
it's fine I will use the terrylinla git for my project for now. But I'm hoping sometimes in the future this can be supported on both ios and android because it is better performing.

**Comment by ShaMan123** [comment]
Yeah, it's much more peeformant. It was the main reason I forked. Android
will run out of memory in the original repo because it uses bitmaps badly.

בתאריך יום ה׳, 21 בינו׳ 2021, 00:13, מאת Cole L. ‏<notifications@github.com

**Comment by colelogan** [comment]
oh that's interesting, That Explains why my android emulator framerate drops when I draw alot of paths. You said that IOS support can be easily implemented what do you do for that?

**Comment by ShaMan123** [comment]
The frame rate drop is a different issue caused because of crossing the
bridge for every movement.
That was solved ysing reanimated. And could be better solved natively.
Regatding iOS there's a dedicated issue listing what needs to be done.

בתאריך יום ה׳, 21 בינו׳ 2021, 12:53, מאת Cole L. ‏<notifications@github.com

**Comment by ShaMan123** [comment]
By the way, I think this kind of ui should be implemented completely in native and pass a single data object at the very end of the interaction.
